### DISCUSSION TITLE
How to blur and disable the background while displaying a clr-spinner in the page center?

### DISCUSSION THREAD

**Discussion by xiao-lix** [vmware/clarity#6723] [Ideas]
Hi, I have a `<form>` element on a page with a couple of `<clr-stepper-panel>` inside. I'm trying to add a spinner on form submit until it jumps to the new page. 

Is there any flag / API clarity offers to blur and disable the background to stop user interactions? 

Thanks for the help.

**Comment by bbogdanov** [comment]
Hi @xiao-lix,

There is no such functionality on the stepper at this time.

What I would do is set a `<div>`  on the top of the stepper panel content and play around with CSS and some state management to accomplish that. I believe it won’t take a long time to implement such a functionality in your product.

I'll move that thread to ideas when implementing the stepper in Clarity Core.
